test/attr/attr_downgrade_exhaustivity.swift

```diff
@@ -0,0 +1,43 @@
+// RUN: %target-typecheck-verify-swift -swift-version 3 %s
+
+enum Runcible {
+  case spoon
+  case hat
+  @_downgrade_exhaustivity_check
```

Comment:
Your comments made me realize I left out half of your test cases from before.  Let me add them back.